### ISSUE TITLE
Update to work with latest  release of event ticket plus.

### ISSUE THREAD

**Issue by jaakkovaakko** [bastiW/event-ticket-plus-shortcode#3]
Hi there!

Have you any plan to update the code to work with the latest release of event ticket plus?

**Comment by bastiW** [comment]
No! As I don’t maintain this project anymore, because of no time. If you are willing to contribute, I can make you admin for this project.
